README.md

```diff
@@ -37,6 +41,7 @@ docker run --rm \
   -e PLUGIN_PIPELINE=drone \
   -e PLUGIN_ENVIRONMENT=production \
   -e PLUGIN_STATE=successful \
+  -e PLUGIN_ADDITIONAL_MESSAGE="DRONE-43 updated the plugin" \
```

Comment:
The request is to introduce a new parameter, `PLUGIN_ISSUEKEYS`, not `PLUGIN_ADDITIONAL_MESSAGE`, which will accept an array of issue keys as shown below, while keeping the existing logic intact:

```
- name: deployment
  image: plugins/jira
  settings:
    client_secret: your-client-secret
    project: SCRUM
    issuekeys:
      - SCRUM-1
      - SCRUM-2
      - SCRUM-3
      - SCRUM-4
```

Could you please address the requirement as mentioned in the issue here: https://github.com/drone/drone-jira/issues/21